packages/babel-parser/test/fixtures/estree/import-assertions/valid-syntax-with-attributes/output.json

```diff
@@ -0,0 +1,54 @@
+{
+  "type": "File",
+  "start":0,"end":51,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":51}},
+  "errors": [
+    "SyntaxError: 'import' and 'export' may appear only with 'sourceType: \"module\"' (1:0)"
```

Comment:
Are we testing this error here?

Reply:
Nope, I forgot to add sourceType module to options.json😬